syntax/javascript.vim

```diff
@@ -337,7 +337,7 @@ if version >= 508 || !exists("did_javascript_syn_inits")
   HiLink jsSwitchColon          Noise
   HiLink jsClassMethodType      Type
   HiLink jsObjectMethodType     Type
-  HiLink jsClassDefinition      jsFuncName
+  HiLink jsClassDefinition      Constant
```

Comment:
I also disagree with this - Technically a class is a function, and therefore should be linked to a function. We provide the `jsClassDefinition` group so you can change it to another color if you'd like.